articles/quickstart-microsoft-provider-format.md

```diff
@@ -284,83 +284,74 @@ try:
     import pulser
 except ImportError:
     !pip -q install pulser
-    !pip -q install pulser-core
 ```
 
 #### Create a quantum register
 
-1. First, load the required imports:
+1. Fresnel is the first PASQAL commercial neutral atoms quantum computer that you can target for your quantum jobs. To configure your quantum registers, you arrange an array of qubits. FRESNEL provides predefined trap layouts that you must utilize to create your quantum register.
```

Comment:
Is it Fresnel, FRESNEL, or Fresnel1? According to the docs, we have [Fresnel1](https://learn.microsoft.com/en-us/azure/quantum/provider-pasqal#fresnel1)